src/Components/ActivationKey/AdditionalRepositoriesCard.js

```diff
@@ -57,18 +49,19 @@ const AdditionalRepositoriesCard = (props) => {
             BaseOS and AppStream, are always enabled and do not need to be
             explicitly added to the activation key.
           </Text>
-          <ButtonWrapper />
+          <AddAdditionalRepositoriesButton
+            onClick={handleEditAdditionalRepositoriesToggle}
+          />
           <AddAdditionalRepositoriesModal
             isOpen={isEditAdditionalRepositoriesModalOpen}
             handleModalToggle={handleEditAdditionalRepositoriesToggle}
-            keyName={activationKey.name}
             repositories={availableRepositories}
             isLoading={isLoading}
             error={error}
           />
         </TextContent>
         <AdditionalRepositoriesTable
-          repositories={activationKey.additionalRepositories}
+          repositories={activationKey.availableRepositories}
```

Comment:
```suggestion
          repositories={activationKey.additionalRepositories}
```
This piece of data needs to stay what it was